src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -85,8 +86,9 @@ void getLinesTest() {
         //then
         assertThat(lineNames.size()).isEqualTo(2);
         assertThat(lineNames.stream()
```

Comment:
아래와 같이 검증해 볼 수도 있겠네요!
```suggestion        
        Set<String> results = new HashSet<>(lineNames);
        assertThat(results).containsExactly(lineName1, lineName2);
```

Reply:
코드가 훨씬 깔끔해졌네요!
수정하겠습니다 :)